analyses/TX_cd_2020/03_sim_TX_cd_2020.R

```diff
@@ -211,10 +302,19 @@ cli_process_done()
 cli_process_start("Computing summary statistics for {.pkg TX_cd_2020}")
 
 plans <- add_summary_stats(plans, map) %>%
-    mutate(total_cvap = tally_var(map, cvap), .after=total_vap)
+    mutate(total_cvap = tally_var(map, cvap), .after = total_vap)
+
+summary(plans)
+
+plans <- plans %>%
```

Comment:
can we move this up to l. 292 (or combine with that pipe)?

the idea is to thin down to 5,000 before saving the `.rds` file (which has no summary stats) and the stats file (which does).

you should be able to just make the change in the code & then just manually make sure that the objects that get uploaded to dataverse have just 5,000 plans each.